### ISSUE TITLE
[BUG] Memory usage.

### ISSUE THREAD

**Issue by 2E0PGS** [jgraph/drawio#87]
When a tab is left open with a drawing loaded the memory of that chrome tab just keeps going up and up slowly. It starts at around 120,000K and then goes all the way upto a gigabyte. It did take several hours to get upto a Gig mind.

If others could try recreate this and confirm it's not just me that would help.

OS: Ubuntu Linux
Google Chrome Version: 57.0.2987.98 (64-bit)
Draw.io Version: v 6.3.8

**Comment by davidjgraph** [comment]
Where was the diagram stored? Did you have all extensions disabled during this test?

**Comment by Edie-roche** [comment]
This issue is a big problem for my company as we use draw.io and Chrome a lot. Can you please fix this. When I finish this comment I have to explain to my managers manager why they can't get to the architecture confluence page. Can you please bump this priority up. Thanks

**Comment by EH-R** [comment]
Can you please bump up the priority on this fix? My company uses draw.io in Confluence and our main browser is Chrome so we are running into this issue a lot. I am getting none stop complaints about pages not loading when they have draw.io in them. Thanks

**Comment by davidjgraph** [comment]
@EH-R Please provide the evidence from your debugging that the problem relates to memory usage.

**Comment by 2E0PGS** [comment]
> Where was the diagram stored? Did you have all extensions disabled during this test?

It was saved to device. E.g. local machine. 

I cannot recall exactly if I was testing in incognito mode or regular mode in Chrome at the time. I guess I would need to test this again but using incognito.

**Comment by sxt9** [comment]
Recently had this in the desktop app. 
Screenshot below: 
![Draw io_MemoryUse](https://user-images.githubusercontent.com/17098837/103468610-f2a71980-4d28-11eb-983c-422dfea7dcf3.png)

I didn't have any diagrams open. I was not attending my computer for a few hours and came back to this.

**Comment by davidjgraph** [comment]
This thread is specific to the web/browser version of diagrams.net. Anyone seen this issue recently there?

**Comment by QuintusFixlein** [comment]
Hi @davidjgraph, I frequently use the confluence draw.io plugin purchased by my company, and all of a sudden this happened, now I am not able to use the tool at all because I open the diagram to edit, do very little there, meanwhile it bumps the memory usage up to 5GB, then it quits working, also I cannot save my work at all. Tested with MS edge and Google chrome.
![image](https://user-images.githubusercontent.com/3540458/151142095-a47f9ea0-b8e5-4da2-8f88-170e5af7ca88.png)

**Comment by QuintusFixlein** [comment]
Actually, I just found the solution: I have copied an existing drawing over and over again and used it as a basis for further diagrams, and it looks like it loads all the other svg (or whatever) into RAM as well. This has worked for quite a while, but now, after the tenth (or so) copy, the system is reaching its limits. I have now tentatively opened a new drawing and copied in only the existing content, everything works as it should :o)
So no action required, thank you.
